Makefile

```diff
@@ -9,10 +9,11 @@ GIT_REVISION := $(shell git rev-parse HEAD)
 UPTODATE := .uptodate
 
 # Building Docker images is now automated. The convention is every directory
-# with a Dockerfile in it builds an image calls quay.io/weaveworks/<dirname>.
+# with a Dockerfile in it builds an image calls quay.io/cortexproject/<dirname>.
 # Dependencies (i.e. things that go in the image) still need to be explicitly
 # declared.
 %/$(UPTODATE): %/Dockerfile
+	@echo
```

Comment:
What's this change for?

Reply:
I put some new lines in the output of the makefile to make it easier to follow what it was doing.  I like them, but not super attached.  WDYT?

Reply:
Makes sense, thanks! LGTM!!!!